int21.c

```diff
@@ -155,6 +172,29 @@ void int21(uc_engine *uc)
                 break;
             }
 
+        case 0x0a: // buffered keyboard input
+            {
+                uint16_t r_dx, r_ds;
+                uint8_t max_buf = 0;
+                char *buf = NULL;
+                char *str = NULL;   
+
+                uc_reg_read(uc, UC_X86_REG_DX, &r_dx);
+                uc_reg_read(uc, UC_X86_REG_DS, &r_ds);
+                uc_mem_read(uc, r_dx, &max_buf, 1);
```

Comment:
In line 190 i write the string in the address pointed by DS:DX with MK_FP
I've tested this function with this https://gist.github.com/Nico01/189129c99dc54b6c7dcc